lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -12,5 +12,39 @@ def fulfillments(options = {})
       fulfillments = get(:fulfillments, options)
       fulfillments.map { |fulfillment| FulfillmentV2.new(fulfillment.as_json) }
     end
+
+    def move(new_location_id:)
+      body = { fulfillment_order: { new_location_id: new_location_id } }.to_json
+      keyed_fulfillment_orders = keyed_fulfillment_orders_from_response(post(:move, {}, body))
+      load_keyed_fulfillment_order(keyed_fulfillment_orders, 'original_fulfillment_order')
+      keyed_fulfillment_orders
+    end
+
+    def cancel
+      keyed_fulfillment_orders = keyed_fulfillment_orders_from_response(post(:cancel, {}, only_id))
+      load_keyed_fulfillment_order(keyed_fulfillment_orders, 'fulfillment_order')
+      keyed_fulfillment_orders
+    end
+
+    def close
```

Comment:
Should be `def close(message: nil)` according to https://help.shopify.com/en/api/reference/shipping-and-fulfillment/unstable/fulfillmentorder#close